### ISSUE TITLE
Specify dependencies

### ISSUE THREAD

**Issue by NobbZ** [sirulli/yoimages#52]
I currently have trouble getting yoImages working in a dockerized environment.

The container is based on [`php:apache`](https://github.com/docker-library/php/blob/35aedb29009b46af6ea1009c9405d01d3f66968e/7.2/stretch/apache/Dockerfile).

Images are neither cropped on upload (if this is a feature at all), nor am I able to do it manually.

So my question is basically if there is a list of dependencies that need to be installed besides WordPress and PHP.

Which PHP-modules are needed?
Which featureset needs to be compiled into them?
What libraries or executables need to be installed?

**Comment by fagia** [comment]
Hi @NobbZ, the plugin doesn't require any additional server libraries other than those already required by WP itself to handle image uploading and cropping. So make sure that php-gd are installed and that the core WP functionalities of image uploading and cropping are working.

**Comment by NobbZ** [comment]
Yeah, in the meantime (5 minutes ago) we realized that it was a problem with GD finding lib-jpeg.

Still I think you should mention in the README or somewhere else, that a properly configured GD is mandatory.
